src/App.js

```diff
@@ -0,0 +1,67 @@
+import "./styles.css";
+import { Routes, Route } from "react-router-dom";
+import { NavBar } from "./Components/NavBar";
+import { Home } from "./Pages/Home";
+import { VideoPlayer } from "./Pages/VideoPlayer";
+import { SideBar } from "./Components/SideBar";
+import { useData } from "./Context/DataProvider";
+import { useEffect } from "react";
+import axios from "axios";
+import { BottomBar } from "./Components/BottomBar";
+import { History } from "./Pages/History";
+import { Snackbar } from "./Components/SnackBar";
+import { WatchLater } from "./Pages/WatchLater";
+import { LikedVideos } from "./Pages/LikedVideos";
+import { PlayList } from "./Pages/PlayList";
+import Modal from "./Components/Modal";
+import { CreatePlayList } from "./Components/CreatePlayList";
+export default function App() {
+  const {
+    state: { snackbarText },
+    openModal,
+    handleClose,
+    dispatch
+  } = useData();
+  useEffect(() => {
+    (async function () {
+      try {
+        const response = await axios.get("/api/videos");
+        localStorage.setItem("data", JSON.stringify(response.data.videos));
+        const getData = JSON.parse(localStorage?.getItem("data"));
```

Comment:
Hey Kajol, here variable name getData initially signifies that it's a function. If you are storing the data only the please rename variable getData to data.

Reply:
Usually we should add verbs in function names as it signifies the action.

Reply:
You are right I have changed it to data now.

Reply:
Looks good now.